bin/aws-secrets-copy

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh -e
+
+from_base=$AWS_SECRETS_BASE_FROM
+[ -z "$from_base" ] && app=$1
```

Comment:
I'm definitely still a novice on shellscript, so bear with me -- this basically means that if the `$from_base` variable is still null after line 3, then the variable `app` will be set to the first arg that was passed in after the shell script was called? do you mean that to be `from_base`? (looking at the structure below and seeing that `app` gets reassigned on line 18)